tests/python/pants_test/java/test_nailgun_client.py

```diff
@@ -170,34 +166,3 @@ def test_execute_protocolerror_on_execute(self, mock_session, mock_try_connect):
 
     def test_repr(self):
         self.assertIsNotNone(repr(self.nailgun_client))
-
-    @unittest.mock.patch("os.kill", **PATCH_OPTS)
```

Comment:
Of these four tests, it feels like this one would be good to keep? We are expecting `Ctrl+C` to continue to kill pantsd here... it will just do it using the pid that it has from the `Handle`.